runtime/dancebox/src/lib.rs

```diff
@@ -629,11 +621,15 @@ construct_runtime!(
 impl_runtime_apis! {
     impl sp_consensus_aura::AuraApi<Block, NimbusId> for Runtime {
         fn slot_duration() -> sp_consensus_aura::SlotDuration {
-            sp_consensus_aura::SlotDuration::from_millis(Aura::slot_duration())
+            sp_consensus_aura::SlotDuration::from_millis(SLOT_DURATION)
         }
 
         fn authorities() -> Vec<NimbusId> {
-            Aura::authorities().into_inner()
+            pallet_authority_mapping::Pallet::<Runtime>::authority_id_mapping(Session::current_index())
```

Comment:
this should be `AuthorityAssignment(Session::current_index()).orchestrator` AFAICT

Reply:
it's not like we use this runtime API, and we could implement it the way you described too, but just to keep it consistent. What do you think?